### ISSUE TITLE
[Presentation] Presentation Title

### ISSUE THREAD

**Issue by magowan** [cncf/tag-security#1320]
Title: Confidential Containers Project Update

Speakers: To be confirmed ->  likely James Magowan (@magowan), possibly another from Project Steering Committee (https://github.com/confidential-containers/confidential-containers/blob/main/governance.md#members)

Description: Provide an update on Confidential Containers Project (https://github.com/confidential-containers)  and our efforts to date.
Our project feels closer to having bandwidth to collaborate more widely with other CNCF projects on the impact of Confidential Computing.
And as a project we want to look to the future and start to consider applying for incubating status, so would welcome the benefit of experiences other projects have had on a journey to incubating status

Time: How long will the presentation take? (estimate) 45 minutes (includes discussion)

Availability: 
Speakers will likely be European or US based and we can adjust speakers to match availability of a slot to present.

TO DO
- [ ] TAG Representative
- [ ] Schedule date
- [ ] By opening this issue, I, @magowan) acknowledge that the presentation topic and speaker will follow the [presentation guidelines](../CONTRIBUTING.md#present-to-the-tag)

**Comment by magowan** [comment]
Thanks,
I guess we don't have a particular time preference (we have a good spread of possible speakers (from Asia to Europe to US) A key thing we are interested in is making contact with other people/projects who have experience with moving from sandbox to incubating status.

If the audiences for EMEA and US don't overlap too much is presenting to both meetings at some point over next few months a possibility?

**Comment by eddie-knight** [comment]
Hi @magowan, it seems like you're primarily looking to connect with other projects for collaboration opportunities and promotion advice. Am I understanding that right?

**Comment by magowan** [comment]
Yes that is right, @PushkarJ suggested a good place to start would be to give an update presentation at one of the Security TAG meetings.
It has been a while since we last presented to the TAG.

**Comment by magowan** [comment]
Can we go with the EMEA meeting on August 28th?
And figure out the US meeting later?

**Comment by eddie-knight** [comment]
Yep, that works! Sorry the replies dropped off here. I'll update the EMEA agenda to lock you in for Aug 28.

**Comment by magowan** [comment]
For some reason the link to presentation guidelines the issue takes from the template doesn't want to work for me.

I am guessing it should take me to here -> https://github.com/cncf/tag-security/blob/main/CONTRIBUTING.md#present-to-the-tag
Acknowledged on that basis .

**Comment by mrcdb** [comment]
Hi @magowan !

The meeting has recently transitioned to LFX but this is not yet updated in the repository README. This is the link: https://zoom-lfx.platform.linuxfoundation.org/meeting/98348738138?password=70e6a945-563a-491f-8485-ecf7394ec13a

See you on Wed!

**Comment by magowan** [comment]
Thanks for the opportunity to present
[20240827_TagSecurity_CoCoUpdate.pdf](https://github.com/user-attachments/files/16800669/20240827_TagSecurity_CoCoUpdate.pdf)

Slides attached here

**Comment by magowan** [comment]
Summer Holidays obviously impact attendance at meetings, it did feel like we should present to the US time zone meeting to reach more people.
Can we use this issue and schedule a slot for the US time zone meeting?

**Comment by magowan** [comment]
Our community was wondering if it would be possible for someone from TAG Security to join one of our regular weekly calls and share an insight into the role of TAG-Security? Perhaps we can encourage some of our members to be more active in TAG-Security focuses discussions/paper etc as well as better understand the role of TAG Security with the Security Assessments.

Happy to take this idea outside the issue and somewhere more sensible to discuss.

Thanks,

**Comment by magowan** [comment]
Where can I find the recording posted after the call?
Specifically looking for the project mentioned that was currently going through Security Assessment and in parallel through incubation process?  My copy/paste from the chat failed me.

**Comment by mnm678** [comment]
Which Wednesdays are you available to present in the US meeting?

The recordings are available through the [calendar entry](https://zoom-lfx.platform.linuxfoundation.org/meetings/cncf?view=week) for the meeting. 

I'll follow up on slack about getting someone from the TAG to meet with your community.

**Comment by mrcdb** [comment]
> Where can I find the recording posted after the call? Specifically looking for the project mentioned that was currently going through Security Assessment and in parallel through incubation process? My copy/paste from the chat failed me.

The project that I mentioned during the call is OpenFGA, whose incubation application is in [progress](https://github.com/cncf/toc/issues/1287) and whose joint-assessment (in collaboration with this group) was [recently merged](https://github.com/cncf/toc/issues/1287) to this repo

**Comment by magowan** [comment]
> Which Wednesdays are you available to present in the US meeting?
> 
> The recordings are available through the [calendar entry](https://zoom-lfx.platform.linuxfoundation.org/meetings/cncf?view=week) for the meeting.
> 
> I'll follow up on slack about getting someone from the TAG to meet with your community.

Can we present to US call on Wednesday 25th September?

**Comment by eddie-knight** [comment]
Hi @magowan! We have several presentations on the schedule right now as part of TOC applications. 

Considering that you're looking for opportunities to collaborate with other projects on security topics, it may be more simple to attend some other presentations to connect directly with projects— or on the weeks where we don't have presentations, our "check-in" meetings have time on the agenda for general discussion to be raised.

**Comment by magowan** [comment]
> Hi @magowan! We have several presentations on the schedule right now as part of TOC applications.
> 
> Considering that you're looking for opportunities to collaborate with other projects on security topics, it may be more simple to attend some other presentations to connect directly with projects— or on the weeks where we don't have presentations, our "check-in" meetings have time on the agenda for general discussion to be raised.

Only part of our goal was the collaboration, the other part was preparing for applying for incubation status and learning from others regarding incubation status.
When I look here -> https://docs.google.com/document/d/1XDmNG1P1YRnidQJEuZFKufaXaIenNHiY-TdYMlt0Un4/
I can't see what topics are scheduled for upcoming meetings our which are checkin meetings
Where can I get a better view of agenda, content for upcoming meetings?

Thanks

**Comment by mrcdb** [comment]
Hi @magowan ! You can find the agenda for upcoming meetings in the table here: https://docs.google.com/document/d/1XDmNG1P1YRnidQJEuZFKufaXaIenNHiY-TdYMlt0Un4/edit#heading=h.2et92p0 .  You can see upcoming meetings that are labeled as "[Presentation"] or check-ins.

**Comment by mnm678** [comment]
Is there interest in scheduling an update in the next month or so?

**Comment by magowan** [comment]
> Is there interest in scheduling an update in the next month or so?

Always interested :-) 
I think we concluded that the presentation to EMEA meeting was our step on the incubation checklist.
Our next step is to provisionally complete the incubation checklist then look to present/discuss with Security TAG.

So I guess to that end we could close this issue as presented/completed , then we can open another in future to focus on our incubation checklist?
